app/views/admin/republishing/index.html.erb

```diff
@@ -31,13 +31,11 @@
         text: "Past prime ministers",
       },
       {
-        text: button_to("Republish",
-          admin_republishing_republish_past_prime_ministers_index_path,
-          method: :post,
+        text: link_to(sanitize("Republish #{tag.span('Past prime ministers', class: 'govuk-visually-hidden')}"),
```

Comment:
Nice 🙌

Reply:
Might be good to add "page" at the end

Reply:
Maybe not worth it yet, but we could add a helper method for this like on RODA: https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/main/app/helpers/application_helper.rb#L16-L24 (the RODA method would need updating to remove an extra space if used in v5 of the design system)